### PR TITLE
Pdf handling

### PR DESCRIPTION
**Pull request by Seiterle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ HITOBITO_EMAIL=pfnörch@flamberg.ch
 HITOBITO_TOKEN=*******************
 HITOBITO_SERVER=https://db.scout.ch
 HITOBITO_LANG=de -- one of: de, fr, it
+HITOBITO_SENDER=1234
 ```
+`HITOBITO_SENDER` is the personal number of the person who will be the sender of the invoice, usually the treasurer.
```
Comment (Seiterle) [inline README.md:16]: we don't need this in the config (store per bulk by using the authenticated users id)

File: billy/app.py [hunk 2]
```diff
@@ -70,6 +79,18 @@ class Meta:
 bulkInvoiceSchema = BulkInvoiceSchema()
 bulkInvoicesSchema = BulkInvoiceSchema(many=True)
 
+@app.route('/upgrade-db', methods=['POST'])
```
Comment (Seiterle) [inline billy/app.py:82]: we are there public routes for this?

File: README.md (hunk 1 above)
Comment (nicoberling) [reply README.md]: Added clarification

File: billy/app.py (hunk 2 above)
Comment (nicoberling) [reply billy/app.py:87]: Removed routes, added "before_first_request" hook for database upgrades

**Pull request merged**